en/docs/get-started/streaming-quick-start-guide.md

```diff
@@ -1,141 +1,103 @@
 # Quick Start Guide - Streaming Integration
 
-This quick start guide gets you started with the Streaming Integrator (SI), in just 5 minutes.
+Let's get started with WSO2 Streaming Integrator(SI) by running a simple streaming use case in your local environment. 
 
-In this guide, you will download the SI distribution as well as Kafka, and then try out a simple Siddhi application. This Siddhi application consumes messages from a file processes the data, generates an output, and then publishes that output to a Kafka topic.
+### Before you begin...
 
-The following is the outline of this quick start guide.
+1. Install [Oracle Java SE Development Kit (JDK)](http://java.sun.com/javase/downloads/index.jsp) version 11 or 1.8 and set the `JAVA_HOME` environment variable.
+   For more information on setting the `JAVA_HOME` environment variable for different operating systems, see [Setup and Install]({{base_path}}/install-and-setup/install/installing-the-product/installing-api-m-runtime/).
+2. Download the Streaming Integrator and Streaming Integrator Tooling distributions from the [WSO2 Streaming Integrator site](https://wso2.com/integration/streaming-integrator/) and extract them to a location of your choice. Hereafter, the extracted location is referred to as `<SI_HOME>` and `<SIT_HOME>` respectively.<br/><br/>
 
-![Quick Start Guide Outline]({{base_path}}/assets/img/streaming/qsg/quick-start.png)
+### What you'll build
+In this sample scenario, you will aggregate the data relating to the raw material purchases of a sweet production factory and publishers the data to a WebSocket receiver.
+                                
 
-!!! tip "Before you begin:"
-    - Download the Streaming Integrator distribution from [WSO2 Streaming Integrator site](https://wso2.com/integration/streaming-integrator/) and extract it to a location of your choice. Hereafter, the extracted location is referred to as `<SI_HOME>`.<br/><br/>
-    - Download the Kafka broker from [the Apache site](https://www.apache.org/dyn/closer.cgi?path=/kafka/2.3.0/kafka_2.12-2.3.0.tgz) and extract it. This directory is referred to as `<KAFKA_HOME>` from here on.
-
-## Step 1: Start the Streaming Integrator
+### Step 1: Start the Streaming Integrator
 
 To start WSO2 Streaming Integrator, navigate to the `<SI_HOME>/bin` directory from the CLI, and issue the appropriate command based on your operating system:
 
 - **For Linux**: `./server.sh`
 - **For Windows**: `server.bat --run`
 
-## Step 2: Create and deploy a simple Siddhi application
+### Step 2: Start the Streaming Integrator Tooling
+To start WSO2 Streaming Integrator Tooling, navigate to the `<SIT_HOME>/samples/sample-clients/websocket-receiver` directory from the CLI, and issue `ant`.
+This will 
+
+- **For Linux**: `./server.sh`
+- **For Windows**: `server.bat --run`
```

Comment:
There seems to be something missing here. Coz it says issue `ant` and then also gives the correct server startup commands.

Reply:
fixed